lib/helper.js

```diff
@@ -225,6 +226,86 @@ class Helper {
   static isNumber(obj) {
     return typeof obj === 'number' || obj instanceof Number;
   }
+
+  /**
```

Comment:
let's not fallback to `util` - the `helper.promisify` implementation is simple enough for our use case. It's easier to read and maintain a single code path.

```js
/**
 * @param {function(?)} nodeFunction
 * @return {function(?):!Promise<?>}
 */
static promisify(nodeFunction) {
  /**
   * @param {!Array<?>} options
   * @return {!Promise<?>}
   */
  return function(...options) {
    return new Promise(function(fulfill, reject) {
      options.push(callback);
      nodeFunction.call(null, ...options);
      function callback(err, result) {
        if (err)
          reject(err);
        else
          fulfill(result);
      }
    });
  };
}
```

Reply:
By "not fallback to util" you mean using util.promisify.custom symbol?

Reply:
yes, and also the `util.promisify` method.

I'd just stick with the simple shim we already have somewhere.

Reply:
you can use https://npmjs.com/util.promisify if you want a compatible shim.